play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukServiceNavigation.scala.html

```diff
@@ -0,0 +1,93 @@
+} else {
+  <div class="@toClasses("govuk-service-navigation", classes)" data-module="govuk-service-navigation" @toAttributes(attributes)>
+}
+
+  <div class="govuk-width-container">
+    @slots.map(_.start) match {
+      case Some(slotStart) => {@Html(slotStart)}
+      case None => {}
+    }
+    <div class="govuk-service-navigation__container">
+      @if(serviceName.nonEmpty) {
+        <span class="govuk-service-navigation__service-name">
+          @serviceUrl match {
+            case Some(url) => {
+              <a href="@url" class="govuk-service-navigation__link"> @serviceName </a>
+              }
+            case None => {
+              <span class="govuk-service-navigation__text">@{serviceName.get}</span>
+            }
+          }
+        </span>
+      }
+      @if(navigation.nonEmpty || slots.nonEmpty) {
+        <nav aria-label="@if(navigationLabel.nonEmpty){@navigationLabel}else{@menuButtonText.getOrElse(messages("govuk.serviceNavigation.menuButtonText"))}" class="@{toClasses("govuk-service-navigation__wrapper", navigationClasses)}">
+          <button type="button" class="govuk-service-navigation__toggle govuk-js-service-navigation-toggle" aria-controls="@navigationId.getOrElse(messages("govuk.serviceNavigation.navigationId"))"@if(menuButtonLabel.nonEmpty && menuButtonLabel.get != menuButtonText.getOrElse(messages("govuk.serviceNavigation.menuButtonText"))){ aria-label="@menuButtonLabel"} hidden>
+            @menuButtonText.getOrElse(messages("govuk.serviceNavigation.menuButtonText"))
+          </button>
+          <ul class="govuk-service-navigation__list" id="@navigationId.getOrElse(messages("govuk.serviceNavigation.navigationId"))">
+            @slots.map(_.navigationStart) match {
+              case Some(slotNavigation) => {@Html(slotNavigation)}
+              case None => {}
+            }
+            @for(item <- navigation) {
+              <li class="govuk-service-navigation__item@if(item.active || item.current){ govuk-service-navigation__item--active}">
+                @if(item.href.nonEmpty) {
+                  <a @if(item.active || item.current){aria-current="@if(item.current){page}else{true}"} class="govuk-service-navigation__link" href="@item.href" @toAttributes(item.attributes)>
+                }else{
+                  <span class="govuk-service-navigation__text">
+                }
+                  @item.current match {
+                    case true => {<strong class="govuk-service-navigation__active-fallback">@item.content.asHtml</strong>}
+                    case _ => {
+                      @item.active match {
+                        case true => {<strong class="govuk-service-navigation__active-fallback">@item.content.asHtml</strong>}
+                        case _ => {@item.content.asHtml}
+                      }    
+                    }
+                  }
```

Comment:
```suggestion
                  @if(item.current || item.active) {
                    <strong class="govuk-service-navigation__active-fallback">@item.content.asHtml</strong>
                  } else {
                     @item.content.asHtml
                  }
```

Reply:
apologies if I missed something in it

Reply:
in my head I think trying to match structure of the govuk-frontend templates as closely as possible is normally a good default, because it makes updates easier to translate https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/service-navigation/template.njk#L51-L55

Reply:
(at least for conditionals I mean)

Reply:
It turns out, this was all that needed to be done - Updated.